.github/workflows/verify-minimal-template-spec-chain.yml

```
@@ -0,0 +1,139 @@
+name: verify-minimal-template-spec-chain.yml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  merge_group:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  actions: write
+
+jobs:
+  preflight:
+    uses: ./.github/workflows/reusable-preflight.yml
+
+  build-wasm:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+
+      - name: Install srtool-cli
+        run: |
+          cargo install --git https://github.com/chevdor/srtool-cli
+          echo "Using srtool-cli Version:"
+          srtool --version
+
+      - name: Pull srtool docker image
+        run: |
+          srtool pull --engine podman
+
+      - name: Build your runtime binary
+        run: |
+          srtool build --package minimal-template-runtime --runtime-dir ./templates/minimal/runtime
```

I think this picks `podman` by defualt, which runs rootless, and you'll need to do something like:
```suggestion
          srtool build --package minimal-template-runtime --runtime-dir ./templates/minimal/runtime --root
```

Same would be for the parachain job too.